apps/safe-claiming-app/src/App.tsx

```diff
@@ -119,22 +120,24 @@ const App = (): ReactElement => {
 
   const chainConstants = CHAIN_CONSTANTS[safe.chainId]
 
-  const [delegates] = useDelegatesFile()
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [delegates, _, delegatesFileError] = useDelegatesFile()
```

Comment:
```suggestion
  const [delegates, , delegatesFileError] = useDelegatesFile()
```

This should also work